README.md

```
@@ -35,7 +35,7 @@ Simple-indexer requires PostgreSQL as a backend database and [TypeORM](https://g
 
 # How to run
 
-### Developement
+### Development
```

Markdownlint (0.35.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)